src/javaharness/java/arcs/api/SlotProxy.java

```diff
@@ -33,6 +33,6 @@ void render(String content) {
     contentJson.forEach(requestedContentTypes::remove);
     // Slot is considered rendered, if a non-empty content was sent and all requested content types
     // were fullfilled.
-    boolean isRendered = requestedContentTypes.size() == 0 && contentJson.keys().size() == 0;
+    //    boolean isRendered = requestedContentTypes.size() == 0 && contentJson.keys().size() == 0;
   }
```

Comment:
This seems like a bug in our Java implementation, unless it's ancillary / only important while doing dev on the Java PEC?

Reply:
It's considered deadcode by errorprone, this line doesn't do anything, I dunno why its there, so I commented it out.

Reply:
@mmandlis should answer this one

Reply:
it was `this.isRendered = ` until https://github.com/PolymerLabs/arcs/pull/3468. 
`this` is not needed, but the code should update a data member, not a local variable.